packages/aws-cdk/lib/init-templates/v1/app/go/README.md

```diff
@@ -0,0 +1,12 @@
+# Welcome to your CDK Go project!
```

Comment:
Should this include a mention that go support is experimental/dev-preview?

Reply:
Good idea. I'll add

Reply:
DONE